wiki/framework/settings-framework.md

```diff
@@ -49,9 +50,10 @@ The load order for the settings are:
 1. Mod Config
 2. Server Config
 3. Mission Config
-4. Placed Mission Modules
+4. Mission Paramaters
+5. Placed Mission Modules
 
-What this means is that at any the 3 points after the Mod Config it is possible to insert your adjusted settings and force those (optionally). This is a powerful tool for server admins, whom can ensure that everyone is using uniform settings across the board on their server. And it provides mission makers the ability to easily set settings for their mission, without creating a large dependency on ACE3; you do not have to place down mission modules.
+What this means is that at any the 4 points after the Mod Config it is possible to insert your adjusted settings and force those (optionally). This is a powerful tool for server admins, whom can ensure that everyone is using uniform settings across the board on their server. And it provides mission makers the ability to easily set settings for their mission, without creating a large dependency on ACE3; you do not have to place down mission modules.
```

Comment:
Whom is incorrect here. Simple rule of thumb is to check if you can replace whom with him/her. It's correct if that's possible, but incorrect when he/she fits better.
Example here being
`He can ensure that everyone is using uniform settings [...]`
which is who not whom.